server/db/db.js

```diff
@@ -493,20 +492,20 @@ async function getSessionWithSessionId(sessionId, pgClient) {
   return null
 }
 
-async function createSession(buttonId, chatbotState, numButtonPresses, incidentCategory, respondedAt, respondedByPhoneNumber, pgClient) {
+async function createSession(buttonId, chatbotState, numberOfAlerts, incidentCategory, respondedAt, respondedByPhoneNumber, pgClient) {
   try {
     const results = await helpers.runQuery(
       'createSession',
       `
-      INSERT INTO sessions (button_id, chatbot_state, alert_type, num_button_presses, responded_at, incident_category, responded_by_phone_number) 
+      INSERT INTO sessions (button_id, chatbot_state, alert_type, number_of_alerts, responded_at, incident_category, responded_by_phone_number) 
       VALUES ($1, $2, $3, $4, $5, $6, $7)
       RETURNING *
       `,
       [
         buttonId,
         chatbotState,
-        numButtonPresses > 1 ? ALERT_TYPE.BUTTONS_URGENT : ALERT_TYPE.BUTTONS_NOT_URGENT,
-        numButtonPresses,
+        numberOfAlerts > 1 ? ALERT_TYPE.BUTTONS_URGENT : ALERT_TYPE.BUTTONS_NOT_URGENT,
+        numberOfAlerts,
```

Comment:
This is now always going to be sent `numberOfAlerts = 1`, so you could remove the logic `numberOfAlerts > 1 ? ALERT_TYPE.BUTTONS_URGENT : ALERT_TYPE.BUTTONS_NOT_URGENT` if you wanted. No big deal to leave it in if you want.

Reply:
Should this be urgent or not urgent upon creation? I'm going to start a session with 1 alert, so by the existing logic, it should be not urgent.

Reply:
I'm keeping this as ALERT_TYPE.BUTTONS_NOT_URGENT; it changes to ALERT_TYPE.BUTTONS_URGENT after another press.

Reply:
Exactly. NOT_URGENT to start with :+1: